Core/UIColorExtension.swift

```diff
@@ -64,6 +68,10 @@ extension UIColor {
     public static var cornflowerBlue: UIColor {
         return UIColor(red: 103.0 / 255.0, green: 143.0 / 255.0, blue: 255.0 / 255.0, alpha: 1.0)
     }
+    
+    public static var skyBlue: UIColor {
```

Comment:
This is rendering too dark

Reply:
I use was using `cornflowerBlue` by mistake.